romana-install/roles/romana/install-master/templates/config/romana.conf.yml

```diff
@@ -49,6 +49,38 @@ services:
     api:
       host: 0.0.0.0
       port: 9604
+      hooks:
+          - executable: {{ romana_bin_dir }}/agent_proxy_add_policy.sh
+            pattern: /policies
+            method: POST
+            when: before
+            output: /var/log/policies-hook.log
+          - executable: {{ romana_bin_dir }}/agent_proxy_delete_policy.sh
+            pattern: /policies
+            method: DELETE
+            when: before
+            output: /var/log/policies-hook.log
     config:
       lease_file : "/etc/ethers"
       wait_for_iface_try: 6
+  - service: policy
+    api:
+      host: {{ romana_master_ip }}
+      port: 9605
+    config:
+      store: 
+        type: mysql
+        host: 127.0.0.1
+        username: root
+        password: {{ stack_password }}
+        database: policy
+  - service: kubernetesListener 
```

Comment:
Should this get wrapped in a conditional block so it only gets included for kubernetes installs? Eg:
```
{% if stack_type == "kubernetes" %}
...
{% endif %}
```